### PR TITLE
development with docker compose

### LINKED ISSUE #235: Development on Docker

### PR DESCRIPTION
**Pull request by Azuki-bar**
once run `docker compose up` command, developers can develop Kokenwiki in Docker.

## changes
- run `db:create` and `db:migrate` in init containers
- run `javascript:build` in Dockerfile
  - so needs to rebuild container image when fixing javascript or CSS sources.
 

close #235

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+RUN apt-get update -qq \
+    && apt-get install -y \
+    ca-certificates \
+    default-mysql-client \
+    nodejs \
+    npm \
+    && rm -rf /var/lib/apt/lists/*
 RUN gem install bundler -v 2.1.4
-
+RUN npm install -g yarn webpack-cli
```
Comment (otariidae) [inline Dockerfile:16]:
```suggestion
RUN npm install -g yarn
```

I don't think it is necessary to install `webpack-cli` globally

**Review by otariidae — CHANGES_REQUESTED** [review]
LGTM
but we are using Google App Engine Standard Environment for Ruby 2.7 as production environment and it uses Node.js v12.22.12
looks like Node.js version is beyond the main purpose of this PR, so you should revert Node.js version in Dockerfile

File: Dockerfile (hunk 1 above)
Comment (Azuki-bar) [reply Dockerfile]: As you say, webpack cli is not necessary.
I will remove it.

**Review by otariidae — APPROVED** [review]
looks great for me 👍

**Comment by Azuki-bar** [comment]
thanks for your review!

**Pull request merged**